tests/integration/profiling/rule_based_profilers/test_profiler_user_workflows.py

```diff
@@ -404,3 +1174,279 @@ def test_quentin_profiler_user_workflow_multi_batch_quantiles_value_ranges_rule(
+                            "allow_relative_error": "$variables.allow_relative_error",
+                        },
+                        "num_bootstrap_samples": "$variables.num_bootstrap_samples",
+                        "false_positive_rate": "$variables.false_positive_rate",
+                        "round_decimals": 2,
+                    }
+                ],
+                "expectation_configuration_builders": [
+                    {
+                        "expectation_type": "expect_column_quantile_values_to_be_between",
+                        "class_name": "DefaultExpectationConfigurationBuilder",
+                        "module_name": "great_expectations.rule_based_profiler.expectation_configuration_builder",
+                        "column": "$domain.domain_kwargs.column",
+                        "quantile_ranges": {
+                            "quantiles": "$variables.quantiles",
+                            "value_ranges": "$parameter.quantile_value_ranges.value.value_range",
+                        },
+                        "allow_relative_error": "$variables.allow_relative_error",
+                        "meta": {
+                            "profiler_details": "$parameter.quantile_value_ranges.details"
+                        },
+                    }
+                ],
+            }
+        },
+    }
+
+    result = validator.expect_column_quantile_values_to_be_between(
+        column="fare_amount",
+        result_format="SUMMARY",
+        include_config=True,
+        auto=True,
+        profiler_config=custom_profiler_config,
+    )
+    assert not result.success
+
+    value_ranges_expected = [
+        [
+            5.842754275,
+            6.5,
+        ],
+        [
+            8.675167517,
+            9.661311131,
+        ],
+        [
+            13.344354435,
+            15.815389039,
+        ],
+    ]
+
+    value_ranges_computed = result.expectation_config["kwargs"]["quantile_ranges"][
+        "value_ranges"
+    ]
+
+    assert len(value_ranges_computed) == len(value_ranges_expected)
+
+    paired_quantiles = zip(
+        value_ranges_computed,
+        value_ranges_expected,
+    )
+    for value_ranges in list(paired_quantiles):
+        for idx in range(2):
+            np.testing.assert_allclose(
+                actual=value_ranges[0][idx],
+                desired=value_ranges[1][idx],
+                rtol=rtol,
+                atol=atol,
+                err_msg=f"Actual value of {value_ranges[0][idx]} differs from expected value of {value_ranges[1][idx]} by more than {atol + rtol * abs(value_ranges[1][idx])} tolerance.",
+            )
+
+    # Use all batches, loaded by Validator, for estimating Expectation argument values.
+    parameter_builder_batch_request = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+
+    custom_profiler_config = {
```

Comment:
this should be a config type

Reply:
@donaldheppner Again -- this is a custom override, provided by users (who may find plain dictionary more convenient -- at least we have held this pattern for user-facing `batch_request` API).

Reply:
@donaldheppner This got done, based on our conversation.  Thanks!